src/senaite/core/tests/doctests/AnalysisTurnaroundTimeForZeroValues.rst

```diff
@@ -0,0 +1,196 @@
+    Analysis Turnaround Time For Zero Values
+------------------------
+
+Running this test from the buildout directory::
+
+    bin/test test_textual_doctests -t AnalysisTurnaroundTimeForZeroValues
+
+
+Test Setup
+..........
+
+Needed Imports:
+
+    >>> import re
+    >>> from AccessControl.PermissionRole import rolesForPermissionOn
+    >>> from bika.lims import api
+    >>> from bika.lims.api.analysis import get_formatted_interval
+    >>> from bika.lims.api.analysis import is_out_of_range
+    >>> from bika.lims.content.analysisrequest import AnalysisRequest
+    >>> from bika.lims.utils.analysisrequest import create_analysisrequest
+    >>> from bika.lims.utils import tmpID
+    >>> from bika.lims.workflow import doActionFor
+    >>> from bika.lims.workflow import getCurrentState
+    >>> from bika.lims.workflow import getAllowedTransitions
+    >>> from DateTime import DateTime
+    >>> from datetime import timedelta
+    >>> from plone.app.testing import TEST_USER_ID
+    >>> from plone.app.testing import TEST_USER_PASSWORD
+    >>> from plone.app.testing import setRoles
+    >>> from Products.ATContentTypes.utils import DT2dt, dt2DT
```

Comment:
Use functions from `senaite.core.api.dtime` for handling dates and times rather than using AT utils. Remember we are moving away from Archetypes towards Py3

Reply:
Also, please cleanup the unused imports (e.g. `rolesForPermissionOn`, `re`, etc.)